app/forms/transfer/task/conditions_met_task_form.rb

```diff
@@ -0,0 +1,14 @@
+class Transfer::Task::ConditionsMetTaskForm < BaseTaskForm
+  attribute :confirm_all_conditions_met, :boolean
+
+  def initialize(tasks_data, user)
+    @tasks_data = tasks_data
+    @user = user
+    @project = @tasks_data.project
+    super(@tasks_data, @user)
+  end
+
+  def save
+    @project.update(all_conditions_met: confirm_all_conditions_met)
```

Comment:
I would bang this (`update!`) as a rule always bang the save methods inside form objects, so you raise loudly if something is wrong.